apps/rush-lib/assets/rush-deploy-init/scenario-template.json

```diff
@@ -0,0 +1,86 @@
+/**
+ * This configuration file defines a deployment scenario for use with the "rush deploy" command.
+ * The scenario files are located under the "common/config/deploy-scenarios" folder.
+ * For full documentation, please see https://rushjs.io
+ */
+ {
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/deploy-scenario.schema.json",
+
+  /**
+   * Specify the name of the main Rush project to be deployed, using the full package name as declared in rush.json.
+   * Dependencies are automatically included with the main project.  If you want the deployment to include
+   * other unrelated projects, add your main project to the "projectSettings" section, and then add the other
+   * projects to its "additionalProjectsToInclude" list.
+   *
+   * The "deploymentProjectNames" array normally contains exactly one item.  However, if "enableSubdeployments"
+   * is true, then multiple projects can be specified, and each item create a separate subdeployment.
+   */
+  "deploymentProjectNames": [
+    "[%PROJECT_NAME_TO_DEPLOY%]"
+  ],
+
+  /**
+   * Subdeployments are an optional feature that enables multiple deployments to be processed with a single
+   * invocation of "rush deploy". In this mode, more than one item can be specified for "deploymentProjectNames".
+   * For each item, a subdeployment folder will be created under the "rush deploy" target folder.
+   * (The folder name can be customized using the "subdeploymentFolderName" project setting.)
+   * Each subdeployment folder will receive its own copy of the named project and its associated dependencies.
+   *
+   * The default value is false.
+   */
+  "enableSubdeployments": false,
```

Comment:
> Wouldn't this be implicitly set to `true` if multiple projects are specified in `deploymentProjectNames`?

Yes, but there are a couple reasons why I wanted it to be separate setting:
- When subdeployments are enabled, the `common/deploy` gets an extra level of folders. This is true even if there is only one item.  Magically changing the folder depth seemed like it would be confusing, e.g. I comment out # 2, and then # 1 suddenly doesn't deploy correctly because its folder disappeared.
- This subtle distinction is easier to understand if we give it an explicit name. "Is this repo using subdeployments or not?"  That setting expresses your intent.

Reply:
> Also shouldn't this by default be commented out and set to `true`?

It could be, but then to answer the question "Is this repo using subdeployments or not?" you would need to read the comment and understand the default.  So maybe it's better for it to be uncommented so there is no question.

The other optional settings are less consequential, because they don't alter your entire folder structure.